### PR TITLE
Initial update

### PR DESCRIPTION
**Pull request by dbnorth**

### CODE REVIEW

File: src/components/admin/maintain/pieces/MaintainPieceCard.vue [hunk 1]
```diff
@@ -2,7 +2,12 @@
 import { ref } from "vue";
 import PieceDialogBody from "./PieceDialogBody.vue";
 import PieceDataService from "../../../../services/PieceDataService.js";
+import { useLoginStore } from "../../../../stores/LoginStore.js";
 
+const loginStore = useLoginStore();
+const isAdmin = ref(loginStore.currentRole.roleId === 3 ? true : false);
+console.log(isAdmin);
```
Comment (JuliaFlores21) [inline src/components/admin/maintain/pieces/MaintainPieceCard.vue:9]: this console.log should me removed

File: src/components/admin/maintain/pieces/MaintainPieceCard.vue [hunk 2]
```diff
@@ -2,7 +2,12 @@
...
+console.log(loginStore.currentRole.roleId);
```
Comment (JuliaFlores21) [inline src/components/admin/maintain/pieces/MaintainPieceCard.vue:10]: this console.log should me removed

File: src/components/admin/maintain/pieces/MaintainPieces.vue [hunk 3]
```diff
@@ -4,6 +4,11 @@ import PieceDataService from "./../../../../services/PieceDataService";
 import ComposerDataService from "./../../../../services/ComposerDataService";
 import MaintainPieceCard from "./MaintainPieceCard.vue";
 import PieceDialogBody from "./PieceDialogBody.vue";
+import { useLoginStore } from "../../../../stores/LoginStore.js";
+
+const loginStore = useLoginStore();
+const isAdmin = ref(loginStore.currentRole.roleId === 3 ? true : false);
+console.log(isAdmin);
```
Comment (JuliaFlores21) [inline src/components/admin/maintain/pieces/MaintainPieces.vue:11]: this console.log should me removed

File: src/components/admin/maintain/pieces/PieceDialogBody.vue [hunk 4]
```diff
@@ -20,7 +19,8 @@ const props = defineProps({
   pieceData: { type: [Object], required: true },
   piecesData: { type: [Array] },
 });
-
+console.log("isAdmin:" + props.isAdmin);
```
Comment (JuliaFlores21) [inline src/components/admin/maintain/pieces/PieceDialogBody.vue:22]: this console.log should me removed

File: src/components/admin/maintain/pieces/PieceDialogBody.vue [hunk 5]
```diff
@@ -20,7 +19,8 @@ const props = defineProps({
...
+console.log("isEdmin:" + props.isEdit);
```
Comment (JuliaFlores21) [inline src/components/admin/maintain/pieces/PieceDialogBody.vue:23]: this console.log should me removed

**Review by JuliaFlores21 — CHANGES_REQUESTED** [review]
Everything works fine. We just need to remove the console.logs

File: src/components/admin/maintain/pieces/MaintainPieceCard.vue (hunk 1 above)
Comment (dbnorth) [reply src/components/admin/maintain/pieces/MaintainPieceCard.vue:9]: done

File: src/components/admin/maintain/pieces/MaintainPieces.vue (hunk 3 above)
Comment (dbnorth) [reply src/components/admin/maintain/pieces/MaintainPieces.vue:11]: done

File: src/components/admin/maintain/pieces/PieceDialogBody.vue (hunk 5 above)
Comment (dbnorth) [reply src/components/admin/maintain/pieces/PieceDialogBody.vue:23]: done

**Review by JuliaFlores21 — APPROVED** [review]
looks good

File: src/components/admin/maintain/pieces/MaintainPieceCard.vue (hunk 2 above)
Comment (dbnorth) [reply src/components/admin/maintain/pieces/MaintainPieceCard.vue]: done

File: src/components/admin/maintain/pieces/PieceDialogBody.vue (hunk 4 above)
Comment (dbnorth) [reply src/components/admin/maintain/pieces/PieceDialogBody.vue]: done

**Pull request merged**